.goreleaser.yml

```diff
@@ -1,14 +1,9 @@
 project_name: osdctl
-
-before:
-  hooks:
-    - go mod tidy
```

Comment:
I removed this primarily because the PR checks were failing on this, but also it's captured in `make` and `make mod` already